contracts/interfaces/IRibbon.sol

```diff
@@ -8,6 +8,12 @@ interface IStrikeSelection {
         returns (uint256, uint256);
 
     function delta() external view returns (uint256);
+
+    function getSimpleStrikePrice(
+        uint256 expiryTimestamp,
+        bool isPut,
+        uint256 strikeMultiplier
+    ) external view returns (uint256, uint256);
```

Comment:
We should just stick to the `getStrikePrice` function because it shares a common interface with the delta based strike selection